main.ts

```diff
@@ -53,4 +54,8 @@ const ctx = await ServerContext.fromManifest(manifest, {
 
 const handler = withLog(ctx.handler());
 
+// Initialize Wasm in the emit module in the background
+// so as not to interfere with server processing.
+tryInstantiateEmitLibWasm();
+
 serve(handler);
```

Comment:
The startup time problem pointed out by https://github.com/denoland/dotland/pull/1902#issuecomment-939899976 is avoided by initializing wasm in the background.
(The log says `isolate start time: 251.75 ms`)